src/extension/features/toolkit-reports/pages/spending-by-payee/component.tsx

```diff
@@ -159,12 +186,14 @@ export class SpendingByPayeeComponent extends React.Component {
           color: PIE_CHART_COLORS[payeeIndex % PIE_CHART_COLORS.length],
           events: {
             click: (event) => {
-              showTransactionModal(event.point.name, event.point.transactions);
+              const point = event.point as Point;
+              showTransactionModal(point.name, point.transactions);
             },
           },
           id: payeeId,
           name: payeeName,
-          transactions: spendingData.get('transactions'),
+          // @ts-ignore We can attach arbitrary data, but Highchart types doesn't support this
+          transactions: spendingData.transactions,
```

Comment:
Prefer `@ts-expect-error` over `@ts-ignore`

Even better would be to correct the type `seriesData` to one that accepts this without having to ignore this line in the compiler.

Reply:
The reason to prefer `@ts-expect-error` is because the compiler will give a warning if there is no longer a compiler error to ignore. This is good for points in the future where the typing improves and the `ts-ignore` unnecessarily shadows type checking (which can lead to other bugs further downstream of the ignored type)